tests/test_grid3d/test_grid.py

```diff
@@ -333,7 +332,6 @@ def test_roffbin_import_v2_banal():
                 assert np.allclose(np.array(xx1), np.array(xx4)) is True
 
 
-@tsetup.bigtest
 def test_roffbin_import_v2stress():
     """Test roff binary import ROFF using new API, comapre timing etc."""
     t0 = xtg.timer()
```

Comment:
Will this run always now and slow testing?

Reply:
The test takes about 1.6s now.